extra_tests/snippets/aot.py

```
@@ -0,0 +1,241 @@
+
+
+assert fib_iter(10) == 55
+# Overflowing partway through is where the compiled loop hands back to the
+# interpreter, which finishes it as a bignum.
+assert fib_iter(95) == 31940434634990099905
+
+
+# Automatic compilation reads annotations, which under PEP 649 means running
+# `__annotate__`. A name that is not defined yet raises there, and that is
+# none of the program's business: it never asked for its annotations.
+def forward(a: NotDefinedYet) -> int:
```

Flake8 (7.3.0)

[error] undefined name 'NotDefinedYet'

(F821)